src/js/components/shepherd-text.svelte

```diff
@@ -24,6 +24,8 @@
     color: rgba(0, 0, 0, 0.75);
     font-size: 1rem;
     line-height: 1.3em;
+    max-height: 30vh;
```

Comment:
Should we make this different for desktop/mobile or do you think this adequately works for most cases? I think I might prefer a max height of something like `75%`.

Reply:
it kind of looks weird on desktop, with the max width of 400px, but I guess that's subjective.

Reply:
also, it covers the element being pointed to, e.g.,

![image](https://user-images.githubusercontent.com/706378/79618615-5a480680-80bf-11ea-8ea3-d8478008052d.png)